packages/icons/web/v1/minus_small.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path d="M9 7h6a1 1 0 1 1 0 2H1a1 1 0 0 1 0-2h8z" fill="#666"/>
```

Comment:
Monochromatic icons should use the `currentColor` value by conventions. This makes it easier to style them with CSS. 

```suggestion
    <path d="M9 7h6a1 1 0 1 1 0 2H1a1 1 0 0 1 0-2h8z" fill="currentColor"/>
```

Reply:
Yep, I will update the asset file.